src/main/java/seedu/application/logic/parser/ArchiveCommandParser.java

```diff
@@ -6,18 +6,27 @@
 import seedu.application.logic.commands.ArchiveCommand;
 import seedu.application.logic.parser.exceptions.ParseException;
 import seedu.application.logic.parser.exceptions.ParseIntegerOverflowException;
+import seedu.application.logic.parser.exceptions.ParsePrefixException;
 
 /**
  * Parses input arguments and creates a new ArchiveCommand object.
  */
 public class ArchiveCommandParser implements Parser<ArchiveCommand> {
 
+    private static final int NUMBER_OF_PREFIX_REQUIRED = 0;
+
     /**
      * Parses the given {@code String} of arguments in the context of the ArchiveCommand
      * and returns a ArchiveCommand object for execution.
      * @throws ParseException if the user input does not conform the expected format.
      */
     public ArchiveCommand parse(String args) throws ParseException {
+
+        if (ArgumentTokenizer.prefixFound(args).size() > NUMBER_OF_PREFIX_REQUIRED) {
```

Comment:
Perhaps `!ArgumentTokenizer.prefixFound(args).isEmpty()` might be a more straightforward way of writing this? Same comment for other parsers with no prefixes like `delete`

Reply:
Yes, I agree, thanks for the suggestion! 👍